### PR TITLE
runtime-config-linux: Add info to clarify oom_score_adj (carrying #236)

### PR DESCRIPTION
**Pull request by vbatts**
This is carrying #236, but deferring the description to the linux docs.

### CODE REVIEW

File: runtime-config-linux.md [hunk 1]
```diff
@@ -180,7 +180,13 @@ For more information, see [the memory cgroup man page](https://www.kernel.org/do
 
 #### Set oom_score_adj
 
-More information on `oom_score_adj` available [here](https://www.kernel.org/doc/Documentation/filesystems/proc.txt).
+oom_score_adj works with disableOOMKiller.
+If out-of-memory killer disabled("disableOOMKiller": true), the value of oom_score_adj is useless and will be omitted.
```
Comment (vishh) [inline runtime-config-linux.md]: https://github.com/opencontainers/specs/pull/236#issuecomment-152684866

Comment (vbatts) [inline runtime-config-linux.md]: fixed

File: runtime-config-linux.md [hunk 2]
```diff
@@ -180,7 +180,14 @@ For more information, see [the memory cgroup man page](https://www.kernel.org/do
 
 #### Set oom_score_adj
 
-More information on `oom_score_adj` available [here](https://www.kernel.org/doc/Documentation/filesystems/proc.txt).
+oom_score_adj sets heuristic regarding how the process is evaluated by the kernel during memory pressure.
```
Comment (wking) [inline runtime-config-linux.md]: “oom_score_adj” should probaby be “**`oom_score_adj`**” or “`oom_score_adj`”.  And maybe reword to:

> **`oom_score_adj`** adjusts the kernel's process evaluation during memory pressure.

File: runtime-config-linux.md [hunk 3]
```diff
@@ -180,7 +180,14 @@ For more information, see [the memory cgroup man page](https://www.kernel.org/do
...
+For more information, see [the proc filesystem documentation section 3.1](https://www.kernel.org/doc/Documentation/filesystems/proc.txt).
+This is a kernel/system level setting, where as disableOOMKiller is scoped for a memory cgroup.
```
Comment (wking) [inline runtime-config-linux.md]: For consistency, I'd backtick/** disableOOMKiller too.

File: runtime-config-linux.md (hunk 2 above)
Comment (vbatts) [inline runtime-config-linux.md]: but it is per process

Comment (vbatts) [inline runtime-config-linux.md]: and i agree, re backticks

**Comment by wking** [comment]
I left a few inline copy-edit suggestions for 1bfadf67, but the thrust
looks good to me.  While we're cleaning up the docs for this setting,
we may want to consider clarifying the runtime action if the setting
is unset or null [1].

[1]: https://github.com/opencontainers/specs/pull/285#issuecomment-167823651

File: runtime-config-linux.md (hunk 2 above)
Comment (wking) [inline runtime-config-linux.md]: On Tue, Jan 05, 2016 at 12:47:42PM -0800, Vincent Batts wrote:
> but it is per process

Right.  I just think “sets heuristic regarding how the process is
evaluated by the kernel” could be condensed without losing
information.  How about something closer to the kernel's §3.1 lead in
[1]:

  `oom_score_adj` adjusts the OOM-killer badness heuristic for this
  process.

?

[1]: https://www.kernel.org/doc/Documentation/filesystems/proc.txt

**Comment by hqhq** [comment]
LGTM

**Comment by philips** [comment]
lgtm

**Comment by vishh** [comment]
LGTM
